assets/aml-benchmark/scripts/custom_inference_postprocessors/humaneval.py

```diff
@@ -172,9 +174,9 @@ def run_humaneval_postprocessor(
         gt = "\n" + row["test"] + "\n" + "check(" + row["entry_point"] + ")"
         if str("def " + row["entry_point"] + "(") in row["original_prediction"]:
             # If the model regenerates the prompt/ function name
-            pred_combined_prompt = row["original_prediction"]
+            pred_combined_prompt = extract_text_from_markdown_tag(row["original_prediction"], tag_type='python')
```

Comment:
Should we also handle the scenario where model does not output python and only the backticks with the code in it (so tag_type is empty string)?

Reply:
This should already be covered by the function - though not used for that in this post-processor